### PR TITLE
rubber 1.4

### PR DESCRIPTION
**Pull request by enilsen16**
Move rubber 1.4 to hombrew-tex per https://github.com/Homebrew/homebrew/pull/47249

### CODE REVIEW

File: Formula/rubber.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+class Rubber < Formula
+  desc "Automated building of LaTeX documents"
+  homepage "https://launchpad.net/rubber/"
+  url "https://launchpad.net/rubber/trunk/1.4/+download/rubber-1.4.tar.gz"
+  sha256 "824af6142a0e52804de2f80d571c0aade1d0297a7d359a5f1874acbb53c0f0b4"
+
+  head "lp:rubber", :using => :bzr
+
+  bottle do
```
Comment (DomT4) [inline Formula/rubber.rb]: `bottle` blocks will need removing. We don't supply bottles for this tap.

**Comment by DomT4** [comment]
Looks like the build is failing here as well.

**Comment by enilsen16** [comment]
I'm not quite sure why pdf compilation is failing with texi2dvi. texinfo should automatically link to the correct path...

Also after a few hours of scratching my head I finally got basictex to install on El Capitan. - https://www.codehive.io/boards/hhjRn-I

**Comment by enilsen16** [comment]
It might be worth adding basictex as an option besides MacTeX.

**Comment by enilsen16** [comment]
Works with Mactex
```
🍺  mactex staged at '/opt/homebrew-cask/Caskroom/mactex/latest' (2.5G)
~ brew install homebrew/homebrew-tex/rubber
==> Installing rubber from homebrew/tex
==> Downloading https://launchpad.net/rubber/trunk/1.4/+download/rubber-1.4.tar.gz
Already downloaded: /Library/Caches/Homebrew/rubber-1.4.tar.gz
==> python setup.py install --prefix=/usr/local/Cellar/rubber/1.4
🍺  /usr/local/Cellar/rubber/1.4: 120 files, 745.1K, built in 3 seconds
```

**Comment by enilsen16** [comment]
Do you have any suggestions @DomT4?

**Comment by DomT4** [comment]
I'm not sure to be honest. This is a bit odd. I'm only running `basictex` locally but it builds fine:
```
==> Summary
🍺  /usr/local/Cellar/rubber/20150624-1.2: 109 files, 496.8K, built in 5 seconds
```

**Comment by enilsen16** [comment]
Well it looks like it fails compiling the pdf documentation with basictex. I think the dependencies have changed from 1.2 - 1.4.

**Comment by dunn** [comment]
I'd be fine leaving them out if @DomT4 is.

**Comment by DomT4** [comment]
Feel free to :skull: 'em. If anyone desperately wants them they're welcome to follow up with a PR reinstating them, providing it passes CI.

**Comment by enilsen16** [comment]
This should work now with basictex.

**Comment by DomT4** [comment]
And merged in https://github.com/Homebrew/homebrew-tex/commit/64f3f16fcc4de7acf6c06c40f275cc6773c71aa0. Thanks again :smiley_cat:

**Pull request closed without merging**